fea.pest

```diff
@@ -469,7 +473,7 @@ genNum = @{ NUMEXT | NUMOCT | NUM }
 
 subtok = @{ SUBSTITUTE | SUBSTITUTE_v }
 revtok = @{ REVERSE | REVERSE_v }
-anontok = @{ ANON | ANON_v }
+anontok = @{ ANON_v | ANON }
```

Comment:
This was the cause of the bug, everything else about this PR just ① makes the test more fair and ② improves our anonymous block handler. But this is why the test failed, because "anon" is tested before "anonymous", and matches "anonymous", then the tag matcher was matching "ymous", and there was no tag. Wham, failure to match.